src/main/java/ladder/domain/LadderResult.java

```diff
@@ -0,0 +1,38 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+class LadderResult {
```

Comment:
일급 컬렉션 👍 복수 명사를 클래스 이름으로 사용하면 컬렉션임을 더 쉽게 표현할 수 있겠죠?

```suggestion
class LadderResults {
```